README.md

```diff
@@ -132,15 +132,16 @@ The process to develop, test and deploy new implementations are as follow:
 2. Bump the version accordingly. Contract versioning should follow semver and increment with each upgrade or deployment.
 3. Develop according to good practices, modify __gap length to accommodate new state, add comments with storage accounting in contract variables and __gap values.
 4. Write an upgradeability test in `test/components/upgrades.test.js`.
-4.1 Deploy an upgradeable proxy with the base implementation, found in `components/_old/<component_folder>/<ComponentName>_X_Y_Z`. 
-    If not there, either:
-    - Use `hardhat flattener` to generate one.
-    - If the contracts is deployed and the repo is not tagged and you are not sure if the exact files are in the current commit, generate a flattened version obtained from verified contracts in relevant block explorer (Etherscan, Polyscan...) and add it.
-    - Rename main component in flattened file to `<ComponentName>_X_Y_Z`.
-4.2 Initialize all the state of the contract, assert the values are there.
-4.3 Upgrade to the new implementation
-4.4 Check the output from `upgrades-plugin`. If gap sizes changed, you may need to activate the flag `unsafeSkipStorageCheck` to deploy.
-4.5 Assert the values of the state have not changed.
+    1 Deploy an upgradeable proxy with the base implementation, found in `components/_old/<component_folder>/<ComponentName>_X_Y_Z`. 
+        If not there, either:
+        - Use `hardhat flattener` to generate a file with all the inheriting contracts.
+        - Remove duplicate licenses and pragma statements.
+        - Rename main component in flattened file to `<ComponentName>_X_Y_Z`.
+        - If the contracts is deployed and the repo is not tagged and you are not sure if the exact files are in the current commit, generate a flattened version obtained from verified contracts in relevant block explorer (Etherscan, Polyscan...) and add it.
+    2 Initialize all the state of the contract, assert the values are there.
+    3 Upgrade to the new implementation
+    4 Check the output from `upgrades-plugin`. If gap sizes changed, you may need to activate the flag `unsafeSkipStorageCheck` to deploy.
```

Comment:
The part about `unsafeSkipStorageCheck` no longer applies right?

Reply:
True! removed it from tests and scripts too aa0cc3e